scapy/contrib/bp.py

```diff
@@ -50,10 +50,13 @@ class BP(Packet):
                    SDNV2('CTSN', 0),
                    SDNV2('LT', 0),
                    SDNV2('DL', 0),
-                   ConditionalField(SDNV2("FO", 0), lambda x: (
-                       x.ProcFlags & 0x01)),
-                   ConditionalField(SDNV2("ADUL", 0), lambda x: (
-                       x.ProcFlags & 0x01)),
+                   MultipleTypeField([
+                       (SDNV2("FO", 0), lambda x: (
+                           x.ProcFlags & 0x01)),
+                       (SDNV2("ADUL", 0), lambda x: (
+                           x.ProcFlags & 0x01)),
+                   ],
```

Comment:
This is incorrect. A MultipleTypeField choses between fields that all have the same name.